### PR TITLE
Misc proc cleanup

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Cleaned up some chest burst stuff with defines, neatened up a few random procs and removed an unused one.
## Why It's Good For The Game
Random cleanup of some code.
## Changelog
:cl:
code: cleaned up some misc code
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon_defines.dm [hunk 1]
```diff
@@ -46,6 +46,8 @@
 	var/datum/action/ability/activable/selected_ability
 	///carbon overlay layers
 	var/list/overlays_standing[TOTAL_LAYERS]
+	///chestburst state
+	var/chestburst = CARBON_NO_CHEST_BURST
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/carbon_defines.dm:50]: whys it on carbon though shouldnt this be human

Comment (Lumipharon) [reply code/modules/mob/living/carbon/carbon_defines.dm:50]: from what I saw this is always referenced at the carbon level. Of course normally only a human can actually be infected

Comment (TiviPlus) [reply code/modules/mob/living/carbon/carbon_defines.dm:50]: Then it should be at the human level, carbons entire existence is stupid

Comment (Lumipharon) [reply code/modules/mob/living/carbon/carbon_defines.dm]: done

**Pull request merged**